pkg/services/alerting/result_handler.go

```diff
@@ -98,6 +100,16 @@ func (handler *defaultResultHandler) handle(evalContext *EvalContext) error {
 		}
 	}
 
-	handler.notifier.SendIfNeeded(evalContext)
+	if err := handler.notifier.SendIfNeeded(evalContext); err != nil {
+		if xerrors.Is(err, context.Canceled) {
+			handler.log.Debug("handler.notifier.SendIfNeeded returned context.Canceled")
+		} else if xerrors.Is(err, context.DeadlineExceeded) {
+			handler.log.Debug("handler.notifier.SendIfNeeded returned context.DeadlineExceeded")
+		} else {
+			// TODO: Consider if error should be propagated
```

Comment:
Remove comment?

Reply:
@marefr Are we sure we don't want to propagate this error?

Reply:
Yes for now at least